test/end2end_test.go

```diff
@@ -7766,3 +7766,86 @@ func (cvd *credentialsVerifyDeadline) Clone() credentials.TransportCredentials {
 func (cvd *credentialsVerifyDeadline) OverrideServerName(s string) error {
 	return nil
 }
+
+func unaryInterceptorVerifyConn(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+	conn := transport.GetConnection(ctx)
+	if conn == nil {
+		return nil, status.Error(codes.NotFound, "connection was not in context")
+	}
+	return nil, status.Error(codes.OK, "")
+}
+
+// TestUnaryServerInterceptorGetsConnection tests whether the accepted conn on
+// the server gets to any unary interceptors on the server side.
+func (s) TestUnaryServerInterceptorGetsConnection(t *testing.T) {
+	for _, e := range listTestEnv() {
+		// Skip this env due to #619.
+		if e.name == "handler-tls" {
+			continue
+		}
+		testUnaryServerInterceptorGetsConnection(t, e)
+	}
+}
+
+func testUnaryServerInterceptorGetsConnection(t *testing.T, e env) {
+	te := newTest(t, e)
```

Comment:
Please don't write new e2e tests using this method.  Use the stub server instead, e.g.

https://github.com/grpc/grpc-go/blob/574137db7de3c10e010d5023626169f13540cef1/test/end2end_test.go#L5337

Reply:
Will do.

Reply:
Changed both tests to use stub server.